app/models/pension/charges/PensionLifetimeAllowancesViewModel.scala

```diff
@@ -29,7 +29,7 @@ case class PensionLifetimeAllowancesViewModel(
                                                pensionAsLumpSumQuestion: Option[Boolean] = None,
                                                pensionAsLumpSum: Option[LifetimeAllowance] = None,
                                                pensionPaidAnotherWayQuestion: Option[Boolean] = None,
-                                               pensionPaidAnotherWay: Option[LifetimeAllowance] = None,
+                                               pensionPaidAnotherWay: LifetimeAllowance = LifetimeAllowance(None, None),
```

Comment:
Switch to merely LifetimeAllowance(), once both values default to 'None'.

Reply:
Change implemented

Reply:
Confirmed.